packages/modular-scripts/src/esbuild-scripts/start/runtime/index.ts

```diff
@@ -107,3 +81,29 @@ connection.onmessage = (m: MessageEvent) => {
     isFirstCompilation = false;
   }
 };
+
```

Comment:
Why has this stuff moved?

Reply:
Lol, me fiddling with it and not putting it back. Doesn't make any difference but will move it again for better clarity.

Reply:
Done